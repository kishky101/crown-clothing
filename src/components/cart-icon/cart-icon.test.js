import renderer from 'react-test-renderer';
import CartIcon from './cart-icon.component';
import { MockStore } from '../../utils/test/setup';


test('creating cart-icon snapshot', () => {
    const carticon = renderer.create(
        <MockStore>
            <CartIcon />
        </MockStore>
        
    ).toJSON;
    expect(carticon).toMatchSnapshot();
})