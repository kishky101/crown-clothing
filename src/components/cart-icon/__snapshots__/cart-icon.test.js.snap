// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creating cart-icon snapshot 1`] = `[Function]`;
